website/static/js/filepage/index.js

```diff
@@ -17,6 +18,53 @@ var Panel = utils.Panel;
 
 var EDITORS = {'text': FileEditor};
 
+var SharePopover =  {
+    view: function(ctrl, params) {
+        var copy_button_height = '34px';
+        var popover_width = '450px';
+        var link = params.link.substring(0, params.link.indexOf('download') + 8) + '%26mode=render';
```

Comment:
This link "cleaning" step should probably happen outside of `SharePopover`, i.e. in `FileViewPage`. Perhaps `SharePopover` should take two params, one for the base URL and one for the iframe src. 

Also, `link` and `url` are ambiguous; these parameter names can be improved. What is the difference between the two?